configuration/analyzer_config.json

```diff
@@ -3196,6 +3196,20 @@
     },
     "params": {}
   },
+  "Virushee_File_Scan":{
+    "type": "file",
+    "python_module": "virushee.Virushee",
+    "description": "Scan a File using VirusHee API",
+    "disabled": false,
+    "external_service": true,
+    "leaks_info": false,
```

Comment:
It can leak info, you are sending a file to an external service...

Reply:
[here](https://github.com/intelowlproject/IntelOwl/issues/440#issue-877100982) it is mentioned that it deletes files after analysis so I thought it should be `false` thank you for clarification ill make necessary changes

Reply:
well, you are still sending the file externally so you lose the control over it. The trust is not enough to be sure that it won't be accessed and used by unwanted people.

Reply:
Alight, Got it 👍🏻